src/plugins/intel_npu/tests/functional/behavior/elf_config.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <base/ov_behavior_test_utils.hpp>
+#include <string>
+#include <vector>
+#include "common/functions.h"
+#include "common/utils.hpp"
+#include "common/npu_test_env_cfg.hpp"
+#include "intel_npu/al/config/common.hpp"
+#include "npu_private_properties.hpp"
+
+namespace {
+
+class ElfConfigTests :
+        public ov::test::behavior::OVPluginTestBase,
+        public testing::WithParamInterface<std::tuple<std::string, ov::AnyMap>> {
+public:
+    void SetUp() override {
+        std::tie(target_device, configuration) = GetParam();
+        OVPluginTestBase::SetUp();
+    }
+
+    static std::string getTestCaseName(testing::TestParamInfo<std::tuple<std::string, ov::AnyMap>> obj) {
+        std::string targetDevice;
+        ov::AnyMap configuration;
+        std::tie(targetDevice, configuration) = obj.param;
+        std::replace(targetDevice.begin(), targetDevice.end(), ':', '.');
+        std::ostringstream result;
+        result << "targetDevice=" << targetDevice << "_";
+        result << "targetPlatform=" << ov::test::utils::getTestsPlatformFromEnvironmentOr(targetDevice) << "_";
+        if (!configuration.empty()) {
+            using namespace ov::test::utils;
+            for (auto& configItem : configuration) {
+                result << "configItem=" << configItem.first << "_";
+                configItem.second.print(result);
+            }
+        }
+        return result.str();
+    }
+
+protected:
+    ov::AnyMap configuration;
+    std::shared_ptr<ov::Core> core = ov::test::utils::PluginCache::get().core();
+};
+
+TEST_P(ElfConfigTests, CompilationWithSpecificConfig) {
+    if (getBackendName(*core) == "LEVEL0") {
```

Comment:
If we are always skipping this test for L0 backend, we do not need it in OpenVINO. We should just check that this property can be set/read in (internal) property tests